hack/local-up-cluster.sh

```diff
@@ -905,9 +905,14 @@ function start_kubedns {
 function start_nodelocaldns {
   cp "${KUBE_ROOT}/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml" nodelocaldns.yaml
   # .* because of the __PILLLAR__ references that eventually will be removed
-  ${SED} -i -e "s/_.*_DNS__DOMAIN__/${DNS_DOMAIN}/g" nodelocaldns.yaml
-  ${SED} -i -e "s/_.*_DNS__SERVER__/${DNS_SERVER_IP}/g" nodelocaldns.yaml
-  ${SED} -i -e "s/_.*_LOCAL__DNS__/${LOCAL_DNS_IP}/g" nodelocaldns.yaml
+  ${SED} -i -e "s/__DNS__DOMAIN__/${DNS_DOMAIN}/g" nodelocaldns.yaml
```

Comment:
Won't this partially replace the string - "__PILLAR__DNS__DOMAIN__"? Can we remove the non-PILLAR lines unless we actually need it?

Reply:
+1 yeah good point haven't tested yet

Reply:
you were right on this, I made sure its ordered properly